### ISSUE TITLE
Add Calendar Entry "Microprofile Website Deployment Pair Programming"

### ISSUE THREAD

**Issue by secondsun** [microprofile/microprofile-marketing#574]
When : Monday September 13 2021 1400-1500 EDT
What : Summers and Cesar are going to pair program / debug deployment of the microprofile website. People are invited to watch.
Where :  ? I think it was mentioned that the zoom channel is available. If so, summers@ajug.org will be the user I log in and that account will need to be able to start the meeting or an admin can join with their zoom skills.

**Comment by aeiras** [comment]
@Emily-Jiang or @jclingan, can you please help add the Zoom event for Website Improvements? Thank you!

@secondsun, 
Once the event is up, you will get access to the Zoom Admin code from either Emily or John, privately. Please keep that code under lock & key and use it in future calls as it is the call used for all MP zoom calls. :) 

Recommendations:

1.  create a website minutes agenda for Website Improvements under the [MP minutes folder ](https://drive.google.com/drive/u/0/folders/0B8_67yGf9LkLZUhOMEFTRWx2TDQ?resourcekey=0-sr7rbZujLQzCbLW1BF9vCg)  Make sure the agenda is added to the zoom calendar event as it is for all other MP calls. 
2. use the [zoom admins guidelines ](https://docs.google.com/document/d/1xTBnhJ2SjDmLjM6OTyRVjSnjb6r5s8_opGjSyZnD5vA/edit) to Steward a MP zoom call
3. Get the MP Zoom Admin code and make sure that the Infra Inventory sheet has you under [Zoom Admins Tab](https://docs.google.com/spreadsheets/d/1bN00rsPpw54CnCjNHHdrty0bvSA5vi1ei7TDXSOf89o/edit#gid=1206744069) 
4. If you wish to push an MP tweet about the call, submit a new ticket with the description of that call and the link to the mp calendar for access. :) 

Thank you for rocking it Summers,

**Comment by aeiras** [comment]
@Emily-Jiang or @jclingan, 

Can you please help with this? Monday is almost here and @secondsun  needs to send a email reminder with the agenda to the MP forum latest tomorrow. He cannot do that without the event being set in the calendar. 

Thank you, all!

**Comment by Emily-Jiang** [comment]
I have scheduled the call and it is on MP calendar as well. Please check whether the slot was correct. You will need to sign into the zoom first in order to join the call @secondsun and Cesar.

**Comment by aeiras** [comment]
Thank you, so much Emily.

Summers, I have provided you with the MP Zoom Admin access that you need to record the call & enter. 
Please remember to send a reminder to our community, so that it might be only you and Cesar attending but call is open for anyone else who might wish to help. :)  Let's do this!

**Comment by secondsun** [comment]
@aeiras

Draft for the reminder email, wdyt

```
All,

Cesar and Summers will be having an hour-long call Monday, September 13 14:00 EDT to discuss the Microprofile Site. During this call we want to focus on uncovering and documenting the current deployment procedures. Hopefully we can begin to unblock wanted improvements to the website. If you are interested in our website infrastructure, lending a hand in the documentation effort, etc please feel free to join us.** 

If you are aware of issues, gaps, etc with the website you want us to be made aware of, feel free to create or comment on an issue on the GitHub tracker : https://github.com/microprofile/microprofile-site/issues . 

Thanks,

MicroProfile

** The meeting link is : https://eclipse.zoom.us/j/86011863158


```

**Comment by secondsun** [comment]
@Emily-Jiang Time slot is perfect. Thanks.

**Comment by aeiras** [comment]
Fantastic stuff, Summers 💪🏼

**Comment by aeiras** [comment]
Summers, I created the Website Improvements minutes https://docs.google.com/document/d/10Ct-upLlvfKoBjQdGasdR-OUsuFhQyVFcdSvbEwC7h4/edit#heading=h.mst7i6gdc7x5 and added it to the calendar for you to Steward. :)

**Comment by cesarhernandezgt** [comment]
Thank you @Emily-Jiang and @secondsun .
@aeiras thank you for the minutes.
I was able to access the link and the timezone is the right one.

Se you all next Monday :)!

**Comment by secondsun** [comment]
Hi guys! Cesar and I have rescheduled the meeting for 1300 PDT / 1600 EDT Thursday the 23rd. Can someone (@aeiras ?) recreate the event?


I will send an email out with the links to the call and minutes

**Comment by aeiras** [comment]
Hola Summers, 

That's good news. 

for future reference, only 3 Admins can help create MP zoom events, Kevins, John, or Emily. : )
Active volunteers are @Emily-Jiang or @jclingan. Can you please help Cesar & Summers match tmw's discussion? 

Thanks, everyone and do add the agenda document to the event. :)
